tailscale/wrapper.sh

```diff
@@ -10,6 +10,51 @@ if [ "${ENABLE_TAILSCALE-}" = true ]; then
     /tailscale/tailscale up --authkey=${TAILSCALE_AUTHKEY} --hostname="${TAILSCALE_APP_NAME}" --accept-routes=true
 fi
 
+export ERL_CRASH_DUMP=/tmp/erl_crash.dump
+
+function upload_crash_dump_to_s3 {
+    EXIT_CODE=${?:-0}
+
+    bucket=$ERL_CRASH_DUMP_S3_BUCKET
+    s3Key=$ERL_CRASH_DUMP_S3_KEY
+    s3Secret=$ERL_CRASH_DUMP_S3_SECRET
+    s3Host=$ERL_CRASH_DUMP_S3_HOST
+    s3Port=$ERL_CRASH_DUMP_S3_PORT
+
+    filePath=${ERL_CRASH_DUMP_FOLDER:-tmp}/$(date +%s)_${ERL_CRASH_DUMP_FILE_NAME:-erl_crash.dump}
+    if [ -f "${ERL_CRASH_DUMP_FOLDER:-tmp}/${ERL_CRASH_DUMP_FILE_NAME:-erl_crash.dump}" ]; then
+        cat ${ERL_CRASH_DUMP_FOLDER:-tmp}/${ERL_CRASH_DUMP_FILE_NAME:-erl_crash.dump} >${filePath}
```

Comment:
Is there a reason to cat the dump file rather than just copy it?
Do you even need to copy it before the upload?

Reply:
it's just to add the timestamp to the filename 😅